.circleci/config.yml

```diff
@@ -18,6 +18,14 @@ jobs:
           name: "Download S1 test data"
           command: |
             python3 tests/download_data.py
```

Comment:
This downloading of real S1 data is taking a long time, and is currently not used anywhere. How about commenting it out for now?

Reply:
Sounds good, I disabled it.